tests/framework/installer/ceph_installer.go

```diff
@@ -93,6 +98,8 @@ func ReturnCephVersion() cephv1.CephVersionSpec {
 		return PacificDevelVersion
 	case "quincy-devel":
 		return QuincyDevelVersion
+	case "reef-devel":
+		return ReefDevelVersion
 	default:
 		return QuincyVersion
```

Comment:
Let's change the default to Reef
```suggestion
		return ReefVersion
```